### PR TITLE
Ensure custom commands are still added to the command list. Fixes #7

### LINKED ISSUE #7: Does not save custom commands to command list
```js
browser.addCommand('george', function() {
    browser.url('/');
})
browser.george();
console.log(browser.getCommandHistory())

// gives:

[ { name: 'init', args: [] },
  { name: 'windowHandleMaximize', args: [] },
  { name: 'timeoutsAsyncScript', args: [ 5000 ] },
  { name: 'url', args: [ '/' ] } ]
```
Custom commands in v4.0 are somehow not added to the `commandList`, so obviously bypass https://github.com/webdriverio/webdriverio/blob/v4.0/lib/webdriverio.js#L169-L172 somehow.

### PR DESCRIPTION
**Pull request by georgecrawford**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -51,7 +52,14 @@ let wrapCommand = function (instance) {
...
             throw new Error(`Command ${fnName} is already defined!`)
         }
-        commandGroup[fnName] = fn
+        commandGroup[fnName] = function() {
+            const name = namespace ? `${namespace}.${fnName}` : fnName
+            instance.commandList.push({
+                name: name,
+                args: arguments
+            })
+            fn.apply(instance, arguments);
```
Comment (georgecrawford) [inline index.js]: @christian-bromann Is `instance` the correct `thisArg` for this call? Or is `null` more appropriate?

**Comment by christian-bromann** [comment]
@georgecrawford is this ready to review?

**Comment by georgecrawford** [comment]
I *think* so. Can't see any problems directly related to this PR at the moment ;)

**Comment by christian-bromann** [comment]
LGTM

**Pull request merged**